File: c/bee1043.c
#include <stdio.h>
#include <stdlib.h>
 
int main() {
    double a, b, c;
    scanf("%lf %lf %lf", &a, &b, &c);

    if((abs(b-c) < a && a < (b+c)) && (abs(a-c) < b && b < (a+c)) && (abs(a-b) < c && c < (a+b))){
        double perimetro = a + b + c;
        printf("Perimetro = %.1lf\n", perimetro);
    } else{
        double area = ((a+b)*c)/2;
        printf("Area = %.1lf\n", area);
    }
    return 0;
}

File: javascript/bee1043.js
var [a, b, c] = require("fs").readFileSync("stdin", "utf8").split(" ").map(Number);

function isTriangle(a, b, c) {
	return Math.abs(b - c) < a && a < b + c && Math.abs(a - c) < b && b < a + c && Math.abs(a - b) < c && c < a + b ? true : false;
}

if (isTriangle(a, b, c)) {
	const perimeter = a + b + c;
	console.log("Perimetro = " + perimeter.toFixed(1));
} else {
	const area = ((a + b) * c) / 2;
	console.log("Area = " + area.toFixed(1));
}
